tests/test_fixtures/expectation_suites/parameterized_expectation_suite_fixture.json

```diff
@@ -16,6 +16,7 @@
         {
             "expectation_type": "expect_column_unique_value_count_to_be_between",
             "kwargs": {
+                "column": "default",
```

Comment:
hmm why did this pop up?

Reply:
`column` is a required param for this expectation, but it was never an issue because it was only ever treated as an expectation config.